orttraining/orttraining/core/session/training_session.cc

```diff
@@ -86,11 +87,9 @@ Status SetupOptimizerParams(
     }
 
     // check if initial optimizer states have been provided for weight
```

Comment:
do we really do the "check" here? seems just retrieve it if there is state passed in.

Reply:
reworded the comment